src/mapgen_functions.cpp

```diff
@@ -152,7 +152,7 @@ void init_mapgen_builtin_functions() {
     mapgen_cfunction_map["basement_spiders"] = &mapgen_basement_spiders;
     mapgen_cfunction_map["office_doctor"] = &mapgen_office_doctor;
     mapgen_cfunction_map["sub_station"] = &mapgen_sub_station;
-    mapgen_cfunction_map["s_garage"] = &mapgen_s_garage;
+//   mapgen_cfunction_map["s_garage"] = &mapgen_s_garage;
```

Comment:
Well, a lot of other entries that got JSONified are *also* commented out. Since I wasn't 'zactly sure why, figured I'd leave that there as well. When I go back through and stomp out this last formatting error I can kill that line easily enough.